### PR TITLE
Add SSL certificate support to Cloud SQL sample app.

### PR DESCRIPTION
**Pull request by jsimonweb**

### CODE REVIEW

File: cloud-sql/postgres/ConnectTcp.cs [hunk 1]
```diff
@@ -40,11 +38,28 @@ public static NpgsqlConnectionStringBuilder NewPostgreSqlTCPConnectionString()
...
             };
+            // [END cloud_sql_postgres_dotnet_ado_connect_tcp]
+            // For deployments that connect directly to a Cloud SQL instance without
+            // using the Cloud SQL Proxy, configuring SSL certificates will ensure the
+            // connection is encrypted.
+            if (Environment.GetEnvironmentVariable("DB_CERT") != null)
+            {
+                connectionString.SslMode = SslMode.VerifyCA;
+                connectionString.SslCertificate =
+                    Environment.GetEnvironmentVariable("DB_CERT"); // 'certs/client.pfx'
```
Comment (amanda-tarafa) [inline cloud-sql/postgres/ConnectTcp.cs:55]:
```suggestion
                    Environment.GetEnvironmentVariable("DB_CERT"); // e.g. 'certs/client.pfx'
```

File: cloud-sql/mysql/ConnectTcp.cs [hunk 2]
```diff
@@ -40,12 +38,29 @@ public static MySqlConnectionStringBuilder NewMysqlTCPConnectionString()
...
             };
+            // [END cloud_sql_mysql_dotnet_ado_connect_tcp]
+            // For deployments that connect directly to a Cloud SQL instance without
+            // using the Cloud SQL Proxy, configuring SSL certificates will ensure the
+            // connection is encrypted.
+            if (Environment.GetEnvironmentVariable("DB_CERT") != null)
+            {
+                connectionString.SslMode = MySqlSslMode.VerifyCA;
+                connectionString.CertificateFile = 
+                    Environment.GetEnvironmentVariable("DB_CERT"); // 'certs/client.pfx'
```
Comment (amanda-tarafa) [inline cloud-sql/mysql/ConnectTcp.cs:55]:
```suggestion
                    Environment.GetEnvironmentVariable("DB_CERT"); // e.g. 'certs/client.pfx'
```

**Review by amanda-tarafa — COMMENTED** [review]
Two tiny nits, and the MS SQL Server readme update is missing? Or is there no need to change the readme in that case?

**Comment by jsimonweb** [comment]
>the MS SQL Server readme update is missing? Or is there no need to change the readme in that case?

Yes no need to change the readme for MS SQL Server at this time.

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Pull request merged**